services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    movie = Movie.objects.all()
+    if genres_ids is not None and actors_ids is not None:
+        movie = movie.filter(genres__id__in=genres_ids,
+                             actors__id__in=actors_ids)
+    if genres_ids is not None:
+        movie = movie.filter(genres__id__in=genres_ids)
+    if actors_ids is not None:
+        movie = movie.filter(actors__id__in=actors_ids)

```

Comment:
You can simplify your code here
```suggestion
    if genres_ids is not None:
        movie = movie.filter(genres__id__in=genres_ids)
    if actors_ids is not None:
        movie = movie.filter(actors__id__in=actors_ids)

```